### ISSUE TITLE
Arrow-head missing in portal / popout

### ISSUE THREAD

**Issue by larissa-n** [pierpo/react-archer#116]
<img width="264" alt="Screen Shot 2020-09-27 at 9 24 27 AM" src="https://user-images.githubusercontent.com/37840830/94366059-c6e71880-00d5-11eb-97fa-1001894839d4.png">

Example: https://codesandbox.io/s/popout-diagram-react-archer-q8y3z

**Comment by pierpo** [comment]
Hi,

Thank you for reporting this along with a reproducing example 😊 
I'll have a look when I find some time, no ETA yet.

Thanks!

**Comment by larissa-n** [comment]
Issue persists with v3.0.0.

**Comment by pierpo** [comment]
Indeed, no changes have been made regarding this. I haven't found the time to solve it. PR welcome if you want to have a look :blush:
